it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/TemplateTestBase.java

```diff
@@ -571,6 +571,11 @@ protected String getGcsPath(String artifactId) {
         artifactBucketName, getClass().getSimpleName(), gcsClient.runId(), artifactId);
   }
 
+  protected String getGcsPath(String artifactId, GcsResourceManager gcsResourceManager) {
+    return ArtifactUtils.getFullGcsPath(
+        artifactBucketName, getClass().getSimpleName(), gcsResourceManager.runId(), artifactId);
```

Comment:
Will this not give `TemplateTestBase.java` as the class name instead of the test class name?

Reply:
No. It gives the actual test class name